src/test/java/de/tum/in/www1/artemis/util/LocalRepository.java

```diff
@@ -0,0 +1,55 @@
+package de.tum.in.www1.artemis.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.apache.commons.io.FileUtils;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.revwalk.RevCommit;
+import org.eclipse.jgit.transport.URIish;
+
+public class LocalRepository {
+
+    public File localRepoFile;
+
+    public File originRepoFile;
+
+    public Git localGit;
+
+    public Git originGit;
+
+    public void configureRepos(String localRepoFileName, String originRepoFileName) throws Exception {
+
+        this.localRepoFile = Files.createTempDirectory(localRepoFileName).toFile();
+        this.localGit = Git.init().setDirectory(localRepoFile).call();
+
+        this.originRepoFile = Files.createTempDirectory(originRepoFileName).toFile();
+        this.originGit = Git.init().setDirectory(originRepoFile).call();
+
+        this.localGit.remoteAdd().setName("origin").setUri(new URIish(String.valueOf(this.originRepoFile))).call();
+    }
+
+    public static void resetLocalRepo(LocalRepository localRepo) throws IOException {
```

Comment:
please convert this into a non static method that is invoked on itself.

Reply:
```suggestion
    public void resetLocalRepo() throws IOException {
```

Reply:
localRepo --> this

Reply:
i adapted the method